pkg/visor/visorconfig/services.go

```diff
@@ -15,9 +15,9 @@ import (
 // Fetch fetches the service URLs & ip:ports from the config service endpoint
 func Fetch(mLog *logging.MasterLogger, serviceConfURL string, stdout bool) (services *Services) {
 
-	serviceConf := fmt.Sprint("http://", serviceConfURL)
+	serviceConf := fmt.Sprint(serviceConfURL)
```

Comment:
Nothing that prevents this from being merged, but this whole line can just be deleted.